service/server.py

```diff
@@ -147,13 +234,32 @@ def get_address_lines(address_data):
     non_empty_lines = [x for x in lines if x is not None]
     return non_empty_lines
 
-#This method attempts to retrieve the index polygon data for the entry
+
+# This method attempts to retrieve the index polygon data for the entry
 def get_property_address_index_polygon(geometry_data):
     indexPolygon = None
     if geometry_data and ('index' in geometry_data):
         indexPolygon = geometry_data['index']
     return indexPolygon
 
-if __name__ == '__main__':
+
+class SigninForm(Form):
+    username = StringField('username', [Required(message='Username is required'), Length(min=4, max=70, message='Username is incorrect')])
+    password = PasswordField('password', [Required(message='Password is required')])
+    
+    def __init__(self, *args, **kwargs):
+        Form.__init__(self, *args, **kwargs)
+
+
+class TitleSearchForm(Form):
+    pass
```

Comment:
This was added for the hidden_tag helper. I also left a TODO saying we should use the WTF form for HTTP form processing and rendering. I'll add a line of comment.